From d98f5626872d7d7bb1cd24a7d7ca376b7f5bd0ee Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Fri, 14 Aug 2015 16:04:19 +0800
Subject: [PATCH 0940/1691] MLK-11370 dts: imx6ul-evk: enhance driver strength
 of sd clk signal

commit d98f5626872d7d7bb1cd24a7d7ca376b7f5bd0ee from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

The SD CLK signal measured by scope shows the signal rise/fall time
does not meet the spec requirement, need increase the pad driver
strength to meet the spec.
The pad setting value is given by the HW team which has been verified
from scope.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
(cherry picked from commit e4944a51c4bb950a8f13e421eefc000a87350e23)
(cherry picked from commit f491804bc93ebd15adb3222ff4774d14eb714491)
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dts b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
index 9a0145e..dfeb7bf 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dts
@@ -481,7 +481,7 @@
 		pinctrl_usdhc1: usdhc1grp {
 			fsl,pins = <
 				MX6UL_PAD_SD1_CMD__USDHC1_CMD     0x17059
-				MX6UL_PAD_SD1_CLK__USDHC1_CLK     0x10059
+				MX6UL_PAD_SD1_CLK__USDHC1_CLK     0x10071
 				MX6UL_PAD_SD1_DATA0__USDHC1_DATA0 0x17059
 				MX6UL_PAD_SD1_DATA1__USDHC1_DATA1 0x17059
 				MX6UL_PAD_SD1_DATA2__USDHC1_DATA2 0x17059
@@ -513,7 +513,7 @@
 
 		pinctrl_usdhc2: usdhc2grp {
 			fsl,pins = <
-				MX6UL_PAD_NAND_RE_B__USDHC2_CLK     0x17059
+				MX6UL_PAD_NAND_RE_B__USDHC2_CLK     0x10069
 				MX6UL_PAD_NAND_WE_B__USDHC2_CMD     0x17059
 				MX6UL_PAD_NAND_DATA00__USDHC2_DATA0 0x17059
 				MX6UL_PAD_NAND_DATA01__USDHC2_DATA1 0x17059
-- 
1.9.1

